resources/META-INF/persistence.xml

```diff
@@ -19,7 +19,26 @@
 		    <property name="show_sql" value="true" />
 		    <!-- Scan for annotated classes and Hibernate mapping XML files -->
 		    <property name="hibernate.archive.autodetection" value="class, hbm"/>
-		    <property name="hibernate.hbm2ddl.auto" value="create" />

```

Comment:
The new persistence unit for seed looks good. Have you tested that the app using the `my-pu` persistence unit works without this line?

Reply:
Yes, that line just tell hibernate to drop all tables and create new ones on start up, everything seems to work without it as long as the database has been seeded